proposals/predict_ml_model_probabilities.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "predict_ml_model_probabilities",
```

Comment:
Like above, this process name looks a bit weird to me.
I'd prefer something like `predict_probabilities`, `ml_model_predict_probabilities` or `ml_predict_probabilities`

Reply:
Thinking about the recent discussion (prefix is based on the primary input), it should probably be `ml_predict_probabilities` (or `ml_model_predict_probabilities` which is rather lon)